packages/sanity/src/core/config/document/actions.ts

```diff
@@ -49,47 +57,55 @@ export interface DocumentActionConfirmDialogProps {
   confirmButtonText?: React.ReactNode
 }
 
-/** @beta */
+/**
+ * @hidden
+ * @beta */
 export interface DocumentActionModalDialogProps {
   type?: 'dialog'
   content: React.ReactNode
   /**
+   *
+   * @hidden
    * @beta
    */
   footer?: React.ReactNode
   /**
+   *
+   * @hidden
    * @beta
    */
   header?: React.ReactNode
   onClose: () => void
   showCloseButton?: boolean
   /**
+   *
+   * @hidden
    * @beta
    */
   width?: 'small' | 'medium' | 'large' | 'full'
 }
 
-/** @beta */
+/**
+ * @hidden
+ * @beta */
 export interface DocumentActionPopoverDialogProps {
   type: 'popover'
   content: React.ReactNode
   onClose: () => void
 }
 
-/** @beta */
```

Comment:
seems like this got wiped during a rebase I am guessing causing build failure

Reply:
Eagle eyes, thank you 🙏

Reply:
Should be re-added now, once more thanks 🙏